front/vite.config.ts

```diff
@@ -154,7 +154,6 @@ export default defineConfig(({ mode }) => {
         INTERCOM_APP_ID: process.env.INTERCOM_APP_ID,
         SENTRY_DSN: process.env.SENTRY_DSN,
         SENTRY_ENV: process.env.SENTRY_ENV,
-        SENTRY_AUTH_TOKEN: process.env.SENTRY_AUTH_TOKEN,
         CI: process.env.CI,
```

Comment:
So if I've understood correctly, we don't actually need to expose this here since we don't need it for global variable replacement (as we don't use the SENTRY_AUTH_TOKEN anywhere during runtime, only while building).

Specifically, we use `SENTRY_AUTH_TOKEN` in the `config.yml`, but there is comes from the circle CI `citizenlab-ee-environment` context.

In the vite.config.ts, we use `process.env.SENTRY_AUTH_TOKEN` directly.